ansible/roles/operator-pipeline/tasks/main.yml

```diff
@@ -77,11 +77,20 @@
   with_fileglob:
     - ../templates/openshift/tasks/*
 
-- name: Deploy pipeline
+- name: Deploy hosted pipeline
   k8s:
     state: present
     apply: yes
     namespace: "{{ oc_namespace }}"
     definition: "{{ lookup('template', '{{ item }}') }}"
   with_items:
     - ../templates/openshift/pipelines/operator-hosted-pipeline.yml
+
+- name: Deploy release pipeline
```

Comment:
You can deduplicate few lines of code by adding a release pipeline to `with_item` section in the generic deploy pipeline task.